test/demo/utils/globalSetup.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
```

Comment:
Somewhat skimming over this setup/puppeteer stuff. Are there any notable differences from what's in FDC3?

Reply:
There are some pretty significant changes to the base puppeteer stuff. Specifically:
 - it is now entirely generic and no longer has any dependencies on the service itself
 - The jestEnv stuff has changed so the puppeteer connection is now handled similarly to the fin connection, i.e. it connects once at the start of the full test run and uses that connection for all tests.
 - The change to jestEnv means we can now use the "mountFunction" util that was previously commented out due to a bug (the jestEnv change was done as an explicit workaround to allow this)

I would **strongly** recommend you fully review these along with the `notificationsRemote` file, as these are pretty much the bulk of the improvements that I'm making in this PR and should be back-ported into FDC3 at some point in the near future.